app/components/user-cards.js

```diff
@@ -59,23 +77,43 @@ export default Component.extend({
 
         this.set('sort_option','first_name')
         this.set('sort_order','asc')
+        this.set('sort_title','First Name')
 
     },
 
+    searchByEmployeeName(){
+        if (this.search !== '') {
+            this.set('displaysearch',this.search);
+            }
+        else {
+            this.set('displaysearch',null);
+        }
+        },   
+
     actions: {
         
         getTeams(teamName) {
             this.set('selected_option',teamName);
-        }  ,
+        },
 
         sortByFields(field){
             this.set('sort_option',field);
+            const title = new Map([
+               [ 'first_name' , "First Name"],
+               ["last_name","Last Name"],
+               ["joiningDate","Joining Date"]
+            ])
+            this.set('sort_title',title.get(field))
```

Comment:
We can achieve this with `sort_option` itself. That's the reason we have `title` property.

Reply:
done